artio-core/src/main/java/uk/co/real_logic/artio/FixCounters.java

```diff
@@ -149,6 +157,36 @@ public AtomicCounter negativeTimestamps()
         return negativeTimestamps;
     }
 
+    public DutyCycleTracker getFramerDutyCycleTracker(final long threshold)
+    {
+        return new DutyCycleStallTracker(
```

Comment:
Maybe in case threshold is zero we could return `DutyCycleTracker` to allow disabling?